data/ecosystems/a/avalanche.toml

```diff
@@ -2218,6 +2219,9 @@ url = "https://github.com/rytell/hero-staking"
 [[repo]]
 url = "https://github.com/sabanstev/ToDoList"
 
+[[repo]]
+url = "https://github.com/salvorio/salvor-contracts"
```

Comment:
no need to add it here if the ecosystem has a subecosystem that has the repo, it operates like a graph and rolls up all the underlying subecosystem repos to compute the attribution later in our ETL, can remove